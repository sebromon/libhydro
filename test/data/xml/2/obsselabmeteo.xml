<?xml version="1.0" encoding="UTF-8"?>
<!-- <hydrometrie xmlns="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1 http://xml.sandre.eaufrance.fr/scenario/hydrometrie/1.1/sandre_sc_hydrometrie.xsd"> -->
<hydrometrie>
  <Scenario>
    <CodeScenario>hydrometrie</CodeScenario>
    <VersionScenario>2</VersionScenario>
    <NomScenario>Echange de données hydrométriques</NomScenario>
    <DateHeureCreationFichier>2010-02-26T23:55:30</DateHeureCreationFichier>
    <RefFichier>meteoplus-rr1_2010-02-26_15-13.xml</RefFichier>
    <Emetteur>
      <CdIntervenant schemeAgencyID="SANDRE">1537</CdIntervenant>
      <CdContact>1</CdContact>
    </Emetteur>
    <Destinataire>
      <CdIntervenant schemeAgencyID="SANDRE">1537</CdIntervenant>
    </Destinataire>
  </Scenario>
  <Donnees>
    <SeriesObsElaborMeteo>
        <SerieObsElaborMeteo>
            <CdSiteMeteo>012345678</CdSiteMeteo>
            <ValPondSiteMeteo>0.51</ValPondSiteMeteo>
            <CdGrdSerieObsElaborMeteo>RR</CdGrdSerieObsElaborMeteo>
            <TypSerieObsElaborMeteo>2</TypSerieObsElaborMeteo>
            <DtDebSerieObsElaborMeteo>2017-04-18T15:16:14</DtDebSerieObsElaborMeteo>
            <DtFinSerieObsElaborMeteo>2017-05-02T08:51:43</DtFinSerieObsElaborMeteo>
            <DureeSerieObsElaborMeteo>60</DureeSerieObsElaborMeteo>
            <SerieObsElaborMeteoIpa>
                <KSerieObsElaborMeteoIpa>0.21</KSerieObsElaborMeteoIpa>
                <PDTSerieObsElaborMeteoIpa>5</PDTSerieObsElaborMeteoIpa>
            </SerieObsElaborMeteoIpa>
            <ObssElaborMeteo>
                <ObsElaborMeteo>
                    <DtObsElaborMeteo>2017-04-19T08:04:44</DtObsElaborMeteo>
                    <ResObsElaborMeteo>10.4</ResObsElaborMeteo>
                    <IndiceQualObsElaborMeteo>88.4</IndiceQualObsElaborMeteo>
                    <QualifObsElaborMeteo>20</QualifObsElaborMeteo>
                    <MethObsElaborMeteo>12</MethObsElaborMeteo>
                    <StObsElaborMeteo>16</StObsElaborMeteo>
                </ObsElaborMeteo>
                <ObsElaborMeteo>
                    <DtObsElaborMeteo>2017-04-20T13:21:08</DtObsElaborMeteo>
                    <ResObsElaborMeteo>11.9</ResObsElaborMeteo>
                </ObsElaborMeteo>
            </ObssElaborMeteo>
        </SerieObsElaborMeteo>
        <SerieObsElaborMeteo>
            <CdSiteHydro>Z7654321</CdSiteHydro>
            <CdGrdSerieObsElaborMeteo>RR</CdGrdSerieObsElaborMeteo>
            <TypSerieObsElaborMeteo>1</TypSerieObsElaborMeteo>
            <DtDebSerieObsElaborMeteo>2011-08-17T19:24:16</DtDebSerieObsElaborMeteo>
            <DtFinSerieObsElaborMeteo>2012-02-08T15:29:31</DtFinSerieObsElaborMeteo>
            <DureeSerieObsElaborMeteo>30</DureeSerieObsElaborMeteo>
            <ObssElaborMeteo>
                <ObsElaborMeteo>
                    <DtObsElaborMeteo>2011-09-01T15:00:00</DtObsElaborMeteo>
                    <ResObsElaborMeteo>28.8</ResObsElaborMeteo>
                    <IndiceQualObsElaborMeteo>33.1</IndiceQualObsElaborMeteo>
                    <QualifObsElaborMeteo>0</QualifObsElaborMeteo>
                    <MethObsElaborMeteo>12</MethObsElaborMeteo>
                    <StObsElaborMeteo>12</StObsElaborMeteo>
                </ObsElaborMeteo>
                <ObsElaborMeteo>
                    <DtObsElaborMeteo>2011-12-15T08:09:10</DtObsElaborMeteo>
                    <ResObsElaborMeteo>38.1</ResObsElaborMeteo>
                </ObsElaborMeteo>
            </ObssElaborMeteo>
        </SerieObsElaborMeteo>
        <SerieObsElaborMeteo>
            <CdSiteHydro>F3214321</CdSiteHydro>
            <CdGrdSerieObsElaborMeteo>RR</CdGrdSerieObsElaborMeteo>
            <TypSerieObsElaborMeteo>1</TypSerieObsElaborMeteo>
            <ObssElaborMeteo>
                <ObsElaborMeteo>
                    <DtObsElaborMeteo>2013-05-04T13:11:26</DtObsElaborMeteo>
                    <ResObsElaborMeteo>55.8</ResObsElaborMeteo>
                </ObsElaborMeteo>
            </ObssElaborMeteo>
        </SerieObsElaborMeteo>
        <!-- serie without pdt IPA -->
        <SerieObsElaborMeteo>
            <CdSiteMeteo>98765432</CdSiteMeteo>
            <ValPondSiteMeteo>0.13</ValPondSiteMeteo>
            <CdGrdSerieObsElaborMeteo>TA</CdGrdSerieObsElaborMeteo>
            <TypSerieObsElaborMeteo>2</TypSerieObsElaborMeteo>
            <SerieObsElaborMeteoIpa>
                <KSerieObsElaborMeteoIpa>0.89</KSerieObsElaborMeteoIpa>
            </SerieObsElaborMeteoIpa>
            <ObssElaborMeteo>
                <ObsElaborMeteo>
                    <DtObsElaborMeteo>2018-03-17T15:23:16</DtObsElaborMeteo>
                    <ResObsElaborMeteo>987.4</ResObsElaborMeteo>
                </ObsElaborMeteo>
            </ObssElaborMeteo>
        </SerieObsElaborMeteo>
    </SeriesObsElaborMeteo>
  </Donnees>
</hydrometrie>
