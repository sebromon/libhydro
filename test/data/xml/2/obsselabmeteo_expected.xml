<?xml version="1.0" encoding="UTF-8"?>
<hydrometrie xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/2" xsi:schemaLocation="http://xml.sandre.eaufrance.fr/scenario/hydrometrie/2 http://xml.sandre.eaufrance.fr/scenario/hydrometrie/2/sandre_sc_hydrometrie.xsd">
<!-- <hydrometrie> -->
  <Scenario>
    <CodeScenario>hydrometrie</CodeScenario>
    <VersionScenario>2</VersionScenario>
    <NomScenario>Echange de données hydrométriques</NomScenario>
    <DateHeureCreationFichier>2010-02-26T23:55:30</DateHeureCreationFichier>
    <Emetteur>
      <CdIntervenant schemeAgencyID="SANDRE">1537</CdIntervenant>
      <CdContact>1</CdContact>
    </Emetteur>
    <Destinataire>
      <CdIntervenant schemeAgencyID="SANDRE">1537</CdIntervenant>
    </Destinataire>
  </Scenario>
  <Donnees>
    <SeriesObsElaborMeteo>
        <SerieObsElaborMeteo>
            <CdSiteMeteo>012345678</CdSiteMeteo>
            <ValPondSiteMeteo>0.51</ValPondSiteMeteo>
            <CdGrdSerieObsElaborMeteo>RR</CdGrdSerieObsElaborMeteo>
            <TypSerieObsElaborMeteo>2</TypSerieObsElaborMeteo>
            <DtDebSerieObsElaborMeteo>2017-04-18T15:16:14</DtDebSerieObsElaborMeteo>
            <DtFinSerieObsElaborMeteo>2017-05-02T08:51:43</DtFinSerieObsElaborMeteo>
            <DureeSerieObsElaborMeteo>60</DureeSerieObsElaborMeteo>
            <SerieObsElaborMeteoIpa>
                <KSerieObsElaborMeteoIpa>0.21</KSerieObsElaborMeteoIpa>
                <PDTSerieObsElaborMeteoIpa>5</PDTSerieObsElaborMeteoIpa>
            </SerieObsElaborMeteoIpa>
            <ObssElaborMeteo>
                <ObsElaborMeteo>
                    <DtObsElaborMeteo>2017-04-19T08:04:44</DtObsElaborMeteo>
                    <ResObsElaborMeteo>10.4</ResObsElaborMeteo>
                    <IndiceQualObsElaborMeteo>88.4</IndiceQualObsElaborMeteo>
                    <QualifObsElaborMeteo>20</QualifObsElaborMeteo>
                    <MethObsElaborMeteo>12</MethObsElaborMeteo>
                    <StObsElaborMeteo>16</StObsElaborMeteo>
                </ObsElaborMeteo>
                <ObsElaborMeteo>
                    <DtObsElaborMeteo>2017-04-20T13:21:08</DtObsElaborMeteo>
                    <ResObsElaborMeteo>11.9</ResObsElaborMeteo>
                    <QualifObsElaborMeteo>16</QualifObsElaborMeteo>
                    <MethObsElaborMeteo>0</MethObsElaborMeteo>
                    <StObsElaborMeteo>0</StObsElaborMeteo>
                </ObsElaborMeteo>
            </ObssElaborMeteo>
        </SerieObsElaborMeteo>
        <SerieObsElaborMeteo>
            <CdSiteHydro>Z7654321</CdSiteHydro>
            <CdGrdSerieObsElaborMeteo>RR</CdGrdSerieObsElaborMeteo>
            <TypSerieObsElaborMeteo>1</TypSerieObsElaborMeteo>
            <DtDebSerieObsElaborMeteo>2011-08-17T19:24:16</DtDebSerieObsElaborMeteo>
            <DtFinSerieObsElaborMeteo>2012-02-08T15:29:31</DtFinSerieObsElaborMeteo>
            <DureeSerieObsElaborMeteo>30</DureeSerieObsElaborMeteo>
            <ObssElaborMeteo>
                <ObsElaborMeteo>
                    <DtObsElaborMeteo>2011-09-01T15:00:00</DtObsElaborMeteo>
                    <ResObsElaborMeteo>28.8</ResObsElaborMeteo>
                    <IndiceQualObsElaborMeteo>33.1</IndiceQualObsElaborMeteo>
                    <QualifObsElaborMeteo>0</QualifObsElaborMeteo>
                    <MethObsElaborMeteo>12</MethObsElaborMeteo>
                    <StObsElaborMeteo>12</StObsElaborMeteo>
                </ObsElaborMeteo>
                <ObsElaborMeteo>
                    <DtObsElaborMeteo>2011-12-15T08:09:10</DtObsElaborMeteo>
                    <ResObsElaborMeteo>38.1</ResObsElaborMeteo>
                    <QualifObsElaborMeteo>16</QualifObsElaborMeteo>
                    <MethObsElaborMeteo>0</MethObsElaborMeteo>
                    <StObsElaborMeteo>0</StObsElaborMeteo>
                </ObsElaborMeteo>
            </ObssElaborMeteo>
        </SerieObsElaborMeteo>
        <SerieObsElaborMeteo>
            <CdSiteHydro>F3214321</CdSiteHydro>
            <CdGrdSerieObsElaborMeteo>RR</CdGrdSerieObsElaborMeteo>
            <TypSerieObsElaborMeteo>1</TypSerieObsElaborMeteo>
            <ObssElaborMeteo>
                <ObsElaborMeteo>
                    <DtObsElaborMeteo>2013-05-04T13:11:26</DtObsElaborMeteo>
                    <ResObsElaborMeteo>55.8</ResObsElaborMeteo>
                    <QualifObsElaborMeteo>16</QualifObsElaborMeteo>
                    <MethObsElaborMeteo>0</MethObsElaborMeteo>
                    <StObsElaborMeteo>0</StObsElaborMeteo>
                </ObsElaborMeteo>
            </ObssElaborMeteo>
        </SerieObsElaborMeteo>
        <SerieObsElaborMeteo>
            <CdSiteMeteo>098765432</CdSiteMeteo>
            <ValPondSiteMeteo>0.13</ValPondSiteMeteo>
            <CdGrdSerieObsElaborMeteo>TA</CdGrdSerieObsElaborMeteo>
            <TypSerieObsElaborMeteo>2</TypSerieObsElaborMeteo>
            <SerieObsElaborMeteoIpa>
                <KSerieObsElaborMeteoIpa>0.89</KSerieObsElaborMeteoIpa>
            </SerieObsElaborMeteoIpa>
            <ObssElaborMeteo>
                <ObsElaborMeteo>
                    <DtObsElaborMeteo>2018-03-17T15:23:16</DtObsElaborMeteo>
                    <ResObsElaborMeteo>987.4</ResObsElaborMeteo>
                    <QualifObsElaborMeteo>16</QualifObsElaborMeteo>
                    <MethObsElaborMeteo>0</MethObsElaborMeteo>
                    <StObsElaborMeteo>0</StObsElaborMeteo>
                </ObsElaborMeteo>
            </ObssElaborMeteo>
        </SerieObsElaborMeteo>
    </SeriesObsElaborMeteo>
  </Donnees>
</hydrometrie>
